src/trello/models.py

```diff
@@ -7,7 +7,8 @@
 
 
 class Project(models.Model):
-	user = models.ForeignKey(User, on_delete=models.CASCADE, null=False, related_name='projects')
+	user = models.ForeignKey(User, on_delete=models.CASCADE, null=True, related_name='projects')
+	manager = models.ManyToManyField(User, null=True, related_name='project', blank=True)
```

Comment:
change related names to smth like owner_projects and manager_projects. Because in your case no difference between projects and project